src/server/api/routes/newGame.router.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const newGameInstanceController = require('../controllers/newGameInstance.controller');
+
+/**
+ * @swagger
+ * /newGame:
+ *   post:
+ *    summary: Create a new game Instance
+ *    description:
+ *      Will create a new game Instance.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: A new game Instance
+ *        description: The new game Instance to create.
+ *        schema:
+ *          type: object
+ *          required:
+ *            - fk_user_id
+ *            - game_timer
+
+ *          properties:
+ *            fk_user_id:
+ *              type: integer
+ *              format: int64 * 
+ *            game_timer:
+ *              type: integer
+ *              format: int64
+ *           
+ *    responses:
+ *      201:
+ *        description: A new Game instance created
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/', (req, res) => {
+  newGameInstanceController
+    .createNewGameInstance(req.body)
+    .then((result) => res.json(result))
+    .catch((error) => {
+      console.log(error);
+
+      res.status(400).send('Bad request').end();
```

Comment:
But having the res. returning a Bad request, means that Swagger response body will always return a 400 Bad request if something goes wrong. 
In this case, that's the only response you will get because you have an error in your insert statement in the controller. 
It can be improved if .catch can return the error to the Swagger body response, so you don't get the standard bad request.

Reply:
you mean here.. should I use middleware with next?